src/PlaywrightEnvironment.ts

```diff
@@ -95,9 +105,21 @@ export const getPlaywrightEnv = (basicEnv = 'node') => {
       this.global.browser = await getBrowserPerProcess(
         playwrightInstance,
         browserType,
-        config,
+        this._jestPlaywrightConfig,
       )
       this.global.context = await this.global.browser.newContext(contextOptions)
+      if (collectCoverage) {
+        ;(this.global.context as BrowserContext).exposeFunction(
```

Comment:
`;` looks wierd

Reply:
ESLint does auto format it like that, since we have disabled the semicolons.